code/modules/projectiles/projectile/bullets.dm

```diff
@@ -148,6 +148,11 @@
 	agony = 1
 	penetration = 2
 
+/obj/item/projectile/bullet/auto380/OnFired() //vector does more damage with regular ammo
+	..()
+	if(ispath(/obj/item/projectile/bullet/auto380, type) && istype(shot_from, /obj/item/weapon/gun/projectile/automatic/vector))
```

Comment:
first of `ispath(/obj/item/projectile/bullet/auto380, type)` is the wrong way around, secondly it will always be true so you can remove that

Reply:
it cuts out any types that aren't base so it only applies to regular ammo
it just works